src/core/toga/widgets/numberinput.py

```diff
@@ -81,7 +85,7 @@ def min_value(self, value):
             raise ValueError("min_value must be an integer")
         except TypeError:
             self._min_value = None
-        self._impl.set_min_value(value)
+        self._impl.set_min_value(self._min_value)
```

Comment:
I think it's now possible for `self._min_value` to not be set by the time you get here, and therefore this call can fail?

Reply:
I don't think so - it's either set on L83, or L87; if a ValueError is raised on L85, L88 won't be executed. I don't see a logic path that ends up with L88 being executed with `_min_value` not set.

Reply:
I agree with @freakboy3742 and also I am not able to find out any scenario in which ```self._min_value``` may not be set. Can you please elaborate on any scenario in which such a case would happen.